Week2/homework/live/exercise.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
This file is not part of the week2 homework and should not be in your pull request.